README.rst

```diff
@@ -204,8 +204,6 @@ To run the all tests run::
 Credits
 =======
 
-* Timing code and ideas taken from: https://bitbucket.org/haypo/misc/src/tip/python/benchmark.py
-
```

Comment:
I think this is the proper link now: https://github.com/vstinner/misc/blob/34d3128468e450dad15b6581af96a790f8bd58ce/python/benchmark.py